backend/src/model/entity/plant_layer.rs

```diff
@@ -12,10 +17,91 @@ use crate::{
     schema::relations,
 };
 
+/// The resolution of the generated heatmap in cm.
+const GRANULARITY: f32 = 10.0;
```

Comment:
Great to have it but it is not used everywhere (not in tests). Why float? Better we use integer everywhere.

Reply:
Float so it is possible to have values between 0 and 1. It is probably not necessary, but doesn't hurt to have.